### ISSUE TITLE
More UI Options

### ISSUE THREAD

**Issue by bennypowers** [vaadin/vaadin-upload#163]
Howdy!
First off - great work. I'm thoroughly impressed with your components.

I'd like to have more control over the UI. Specifically, I want to be able to do something like this:

![screen shot 2017-03-23 at 9 43 07 am](https://cloud.githubusercontent.com/assets/1466420/24236935/d32dd9f8-0fac-11e7-85eb-35d1eeabd1b4.png)

In this example, I'm using something like a `paper-icon-button` instead of a `paper-button` and I've altered the border-radius of the drop zone.

Basically, I'd like for the ` <div id="buttons">` to be a content slot. This would let me build pretty much any upload UI I want

**Comment by stramel** [comment]
Take a look at #148 where I have added the ability to slot the entire component. 

You can also change the `border-radius` on the drop zone by applying the property to `vaadin-upload`

**Comment by bennypowers** [comment]
@stramel nice work!
here are the styles I applied to approximate the mockup above:
```css
vaadin-upload {
  border-radius: 500px !important;
  --primary-color: var(--app-upload-color);
  --divider-color: var(--app-divider-color, #666);
  --vaadin-upload-drop-label: {
    display: flex;
    justify-content: center;
    align-items: center;
    height: calc(100% - 32px);
    font-size: 2rem;
    height: 100%;
    padding: 0;
  }
}
#uploadButton {
  width: 200px;
  height: 200px;
  --iron-icon-height: 100%;
  --iron-icon-width: auto;
  --iron-icon: {
    border-radius: 50%;
    overflow: hidden;
  }
}
.drop-label iron-icon {
  width: 48px;
  height: 48px;
}
```
